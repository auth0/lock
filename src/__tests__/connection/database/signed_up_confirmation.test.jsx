import React from 'react';
import I from 'immutable';

import { expectComponent } from 'testUtils';

import SignedUpConfirmation from '../../../connection/database/signed_up_confirmation';

const lock = I.fromJS({ id: '__lock-id__' });

describe('SignedUpConfirmation', () => {
  it('renders correctly', async () => {
    expectComponent(<SignedUpConfirmation lock={lock} />).toMatchSnapshot();
  });
});
