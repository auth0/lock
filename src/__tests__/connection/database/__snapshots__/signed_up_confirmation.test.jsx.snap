// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignedUpConfirmation renders correctly 1`] = `
<div
  className="auth0-lock-confirmation"
>
  <div
    className="auth0-lock-confirmation-content"
  >
    <span
      dangerouslySetInnerHTML={
        Object {
          "__html": "<svg focusable=\\"false\\" width=\\"56px\\" height=\\"56px\\" viewBox=\\"0 0 52 52\\" version=\\"1.1\\" xmlns=\\"http://www.w3.org/2000/svg\\" xmlns:xlink=\\"http://www.w3.org/1999/xlink\\" class=\\"checkmark\\"> <circle cx=\\"26\\" cy=\\"26\\" r=\\"25\\" fill=\\"none\\" class=\\"checkmark__circle\\"></circle> <path fill=\\"none\\" d=\\"M14.1 27.2l7.1 7.2 16.7-16.8\\" class=\\"checkmark__check\\"></path> </svg>",
        }
      }
    />
    <p />
  </div>
</div>
`;
